schema/objects/Stakeholder.schema.json

```diff
@@ -29,6 +29,10 @@
       "description": "What is the current relationship of the stakeholder to the issuer?",
       "$ref": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/enums/StakeholderRelationshipType.schema.json"
     },
+    "current_status": {
+      "description": "What is the current activity status of the stakeholder?",
+      "$ref": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/enums/StakeholderStatusType.schema.json"
+    },
```

Comment:
added `current_status` to go alongside `current_relationship`

I thought of maybe changing these to be initial values instead, but I don't think that's as useful at a glance, nor would many platforms be able to provide this reliably

Reply:
And the idea here @jacobyavis is that we'd update the stakeholder objects on export to always represent the most recent status? 

Since the change events support multiple relationships to the issuer, shouldn't current_relationship be a list?

Reply:
Good point. I've now introduced `current_relationships` as an array, and marked the singular version with a deprecation warning